plugins/obs-qsv11/QSV_Encoder.cpp

```diff
@@ -109,6 +109,7 @@ bool prefer_igpu_enc(int *iGPUIndex)
 		return false;
 	}
 
+	// Check for i+I cases (default will be integrated)
```

Comment:
Could you please clarify this comment?  What is "i+I"?

Reply:
Clarified to: // Check for i+I cases (Intel discrete + Intel integrated graphics on the same system). Default will be integrated.